src/shaders/background.fragment.glsl

```diff
@@ -1,18 +1,18 @@
 uniform vec4 u_color;
 uniform float u_opacity;
 
-#ifdef FOG
-varying float v_depth;
+#if defined( FOG ) && !defined( RENDER_TO_TEXTURE )
```

Comment:
I think the second might actually make more sense since as far as the shaders are concerned, fog is not active in any way when operating in render-to-texture mode.